openshift/cronJob-template.yaml

```diff
@@ -0,0 +1,140 @@
+                    # Give analyzer 10 minutes to run this job results, kill it if it was not able result anything.
+                    tcpSocket:
+                      port: 80
+                    initialDelaySeconds: 600
+                    failureThreshold: 1
+                    periodSeconds: 10
+                  env:
+                    - name: THOTH_LOG_INIT_JOB
+                      value: "${THOTH_LOG_INIT_JOB}"
+                    - name: KUBERNETES_API_URL
+                      value: https://kubernetes.default.svc.cluster.local
+                    - name: KUBERNETES_VERIFY_TLS
+                      value: "0"
+                    - name: THOTH_RESULT_API_URL
+                      value: http://result-api
+                    - name: THOTH_INIT_JOB_REGISTER_INDEXES_ONLY
+                      value: "true"
+                    - name: THOTH_MIDDLETIER_NAMESPACE
+                      valueFrom:
+                        configMapKeyRef:
+                          name: thoth
+                          key: middletier-namespace
+                    - name: THOTH_INFRA_NAMESPACE
+                      valueFrom:
+                        configMapKeyRef:
+                          name: thoth
+                          key: infra-namespace
+                    - name: SENTRY_DSN
+                      valueFrom:
+                        secretKeyRef:
+                          key: sentry-dsn
+                          name: thoth
+                    - name: THOTH_DEPLOYMENT_NAME
+                      valueFrom:
+                        configMapKeyRef:
+                          key: storage-bucket-name
+                          name: thoth
+                    - name: KNOWLEDGE_GRAPH_HOST
+                      valueFrom:
+                        configMapKeyRef:
+                          key: postgresql-host
+                          name: thoth
+                    - name: KNOWLEDGE_GRAPH_PORT
+                      value: "5432"
+                    - name: KNOWLEDGE_GRAPH_SSL_DISABLED
+                      value: "1"
+                    - name: KNOWLEDGE_GRAPH_USER
+                      valueFrom:
+                        secretKeyRef:
+                          name: postgresql
+                          key: database-user
+                    - name: KNOWLEDGE_GRAPH_PASSWORD
+                      valueFrom:
+                        secretKeyRef:
+                          name: postgresql
+                          key: database-password
+                    - name: KNOWLEDGE_GRAPH_DATABASE
+                      valueFrom:
+                        secretKeyRef:
+                          name: postgresql
+                          key: database-name
+                  resources:
+                    limits:
+                      memory: '256Mi'
+                      cpu: '500m'
+                    requests:
+                      memory: '256Mi'
+                      cpu: '500m'
```

Comment:
There was `test: false` parameter in [job-template](https://github.com/thoth-station/init-job/blob/master/openshift/job-template.yaml#L109) which I deleted here in `cronJob-template`